core/tests/participants.py

```diff
@@ -124,4 +128,23 @@ def test_can_query_maiden_name(self):
 
         self.assertEqual(200, response.status_code)
         self.assertTrue(len(content) == 1)
-        
\ No newline at end of file
+
+    def test_return_four_oh_four(self):
+        headers = self.auth_headers_for_user("front_desk")
+        known_participant_id = 100000
```

Comment:
this is minor but the variable name should be something more like invalid_id or out_of_range_id, something to indicate that its not in the db.

Reply:
Yup no worries! Sorry for being a bit sloppy.

Reply:
its all good. I feel sloppy on my code reviews, I didnt notice it till now.